ow_faults/README.md

```diff
@@ -2,72 +2,6 @@ The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
 Research and Simulation can be found in README.md in the root directory of
 this repository.
 
-ow_faults
+ow_faults_detection
```

Comment:
This file didn't need to be changed ... for the most part

Reply:
oh i updated the wrong readme, will address